js-exercises/parse-ms/parseMs.js

```diff
@@ -0,0 +1,15 @@
+const parseMs = ms => {
+  let parseMsObj = {
+    'days': parseInt(ms/(24*3600000)),
```

Comment:
Avoid naked numbers as it doesn't give any context. In place of directly writing `24*3600000` it can be done like this
`const millisecondsInADay = 24*3600000`.